hack/quickstart/init-worker.sh

```diff
@@ -13,57 +13,41 @@ function usage() {
     exit 1
 }
 
-function extract_master_endpoint (){
```

Comment:
The generated (bootstrap-)kubeconfig already contains the Private IP of the API server. We don't need to do anything else now that we are removing the `--api-servers` parameter: Kubelet gets it from kubeconfig.